server/utils/presenterUtils.ts

```diff
@@ -26,6 +39,41 @@ export default class PresenterUtils {
     return null
   }
 
+  durationValue(
+    modelValue: Duration | null,
+    userInputKey: string,
+    error: FormValidationError | null
+  ): DurationInputPresenter {
+    const [hoursKey, minutesKey] = ['hours', 'minutes'].map(suffix => `${userInputKey}-${suffix}`)
+
+    const errorMessage = PresenterUtils.errorMessage(error, hoursKey) ?? PresenterUtils.errorMessage(error, minutesKey)
+
+    let hoursValue = ''
```

Comment:
Don't know if this matters or not, but I think we can just declare this rather than assigning it to an empty string as it's always assigned in the `else` below, i.e.`let hoursValue: string`

Reply:
It's not assigned if the nested `if (modelValue !== null)` is false, though, so the compiler would complain.

Reply:
ah you're right